tests/vault-py3/main.tf

```diff
@@ -4,7 +4,29 @@ terraform {
 
 resource "random_id" "name" {
   byte_length = 6
-  prefix      = "tf-vault"
+  prefix      = "tf-vault-"
+}
+
+locals {
+#   override_json = <<-OVERRIDE
```

Comment:
This commented out intentionally?

Reply:
yes. Was testing without the override

Reply:
let's not leave things like that in the source. make a separate test to vary the inputs, or call the module twice in this config, once with each set of inputs being tested.

Reply:
ok. updated with